types/koa-joi-router/index.d.ts

```diff
@@ -44,6 +44,7 @@ declare namespace createRouter {
         routes: Spec[];
         route(spec: Spec|Spec[]): Router;
         middleware(): Koa.Middleware;
+        prefix(path: string): string;
```

Comment:
Doesn't return a string, I'm afraid. Looks like https://github.com/koajs/joi-router/blob/master/joi-router.js#L42-L45 means the relevant code is here: https://github.com/alexmingoia/koa-router/blob/master/lib/router.js#L296-L306

So prefix actually returns the private property `router` 😐

Reply:
@wingsbob I have replaced `string` with `Router` type. I checked `@type/koa-router` and it returns `Router` as the type for `prefix`, relevant code https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/koa-router/index.d.ts#L221

Reply:
@martinmicunda Unfortunately, `Router` in koa-router and `Router` in koa-joi-router are different types.

I think the options here are to import the type from `koa-router` to return, or to use the `void` type, as I don't think there's a great use-case for using the return value.

Reply:
@wingsbob ah okay. I have replaced the return type with `void`.